### PR TITLE
wgengine/router: don't use 88 or 8888 as table/rule numbers.

### PR DESCRIPTION
**Pull request by danderson**
We originally picked those numbers somewhat at random, but with the idea
that 8 is a traditionally lucky number in Chinese culture. Unfortunately,
"88" is also neo-nazi shorthand language.

Use 52 instead, because those are the digits above the letters
"TS" (tailscale) on a qwerty keyboard, so we're unlikely to collide with
other users. 5, 2 and 52 are also pleasantly culturally meaningless.

Signed-off-by: David Anderson <danderson@tailscale.com>

---
This change is [Reviewable](https://reviewable.io/reviews/tailscale/tailscale/584)

### CODE REVIEW

**Comment by apenwarr** [comment]
I think this will leave crumbs in the 'ip rule' list after an update, but those should be pretty harmless since the 88xx tables they refer to will be empty anyhow.

File: wgengine/router/router_linux.go [hunk 1]
```diff
@@ -46,6 +46,11 @@ const (
 	tailscaleBypassMark = "0x80000"
 )
 
+// tailscaleRouteTable is the routing table number for Tailscale
+// network routes. See addIPRules for the detailed policy routing
+// logic that ends up doing lookups within that table.
+const tailscaleRouteTable = "52"
```
Comment (bradfitz) [inline wgengine/router/router_linux.go:52]: Note the TS mnemonic here? And mention that it has to be less than 256 for Busybox?

Comment (danderson) [reply wgengine/router/router_linux.go:63]: Done.

**Comment by danderson** [comment]
@apenwarr You're right that this will leave crumbs behind. I was poised to argue with you about whether that's harmless, but then I remembered that linux helpfully flushes routes that point to an interface when that interface goes away, so table 88 will in fact empty out when tailscaled restarts, and the crumbs will be harmless. Spiffy!

**Pull request merged**